README.md

```diff
@@ -1,6 +1,14 @@
 [![FINOS - Incubating](https://cdn.jsdelivr.net/gh/finos/contrib-toolbox@master/images/badge-incubating.svg)](https://finosfoundation.atlassian.net/wiki/display/FINOS/Incubating)
 [![npm version](https://badge.fury.io/js/generator-symphony.svg)](https://badge.fury.io/js/generator-symphony)
 [![CircleCI](https://circleci.com/gh/SymphonyPlatformSolutions/generator-symphony.svg?style=shield)](https://circleci.com/gh/SymphonyPlatformSolutions/generator-symphony)
+
+## ⚠️ Important notice ⚠️
```

Comment:
The README should be https://www.npmjs.com/package/generator-symphony today
But I guess it has not been committed and only changed locally...
Could take that one instead and just add this notice paragraph?

Reply:
Done https://github.com/finos/generator-symphony/pull/187/commits/35a8db10ae807eb1ae24d7c923effc4d52ea1b76